Analysis/backgrounds_spike.tex

```diff
@@ -159,3 +162,5 @@
 This results in a minor change of the value of D of $12.1 \pm 1.7$. Next, A and B are computed using a lower-\pt SinglePhoton sample, requiring triggers \texttt{Photon135\_PFMET100} or \texttt{Photon120\_R9Id90\_HE10\_IsoM} to have fired, instead of the signal trigger. 
 The second modification gives $D=9.1 \pm 1.3$. 
 We then take twice the discrepancy between the nominal D and the D value from the second modification to obtain a 33\% systematic uncertainty on the spike background estimate.
+\PH{Are you not using hte signal region for the spike
+  estimate. Perhaps you can add some text to justify this is ok. }
```

Comment:
once again, table should help with this. but A and B are special sample, and C is the signal region with the trivial shower shape cuts inverted.